### PR TITLE
bump nodejs package dependencies

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
   "devDependencies": {
     "gulp": "~3.9.1",
-    "laravel-elixir": "~5.0.0",
+    "laravel-elixir": "~6.0.0-9",
     "animate-sass": "git+https://github.com/tgdev/animate-sass.git",
     "autosize": "^3.0.15",
     "bootstrap-sass": "^3.3.6",
     "chart.js": "^2.1.2",
-    "eonasdan-bootstrap-datetimepicker": "~3.1",
+    "eonasdan-bootstrap-datetimepicker": "~4.15",
```
Comment (jbrooksuk) [inline package.json]: We don't use v4 of the bootstrap datetimepicker yet, because of some weird positioning issues. Could you revert this change please?

File: package.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
     "github-markdown-css": "^2.3.0",
     "ionicons": "~2.0",
     "jquery-minicolors": "^2.1.10",
     "jquery-serializeobject": "^1.0.0",
     "jquery-sparkline": "^2.3.2",
-    "jquery": "~2.2",
+    "jquery": "~3.1",
```
Comment (jbrooksuk) [inline package.json]: I'm not sure about upgrading to jQuery v3 yet, customers may not be browsing Cachet on a new browser. Although, what's support like?

Comment (PeterDaveHello) [inline package.json]: Okay, let's rollback this one.

File: package.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
   "name": "cachet",
   "devDependencies": {
     "gulp": "~3.9.1",
-    "laravel-elixir": "~5.0.0",
+    "laravel-elixir": "~6.0.0-9",
```
Comment (GrahamCampbell) [inline package.json]: Why the 9?

**Comment by GrahamCampbell** [comment]
We should probably bump elixir on 2.4.x (currently the master branch), rather than on 2.3.x.

**Comment by PeterDaveHello** [comment]
Should I update this PR?

**Comment by jbrooksuk** [comment]
Yes please.

File: package.json (hunk 3 above)
Comment (jbrooksuk) [inline package.json]: Without the `-9`, this happens:

```
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/Cellar/node/6.2.1/bin/node" "/usr/local/bin/npm" "update"
npm ERR! node v6.2.1
npm ERR! npm  v3.10.0
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: laravel-elixir@'>=6.0.0 <6.1.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.3.7, 0.3.8, 0.3.9, 0.3.10, 0.3.11, 0.4.0, 0.4.1, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.4.7, 0.4.8, 0.4.9, 0.4.10, 0.4.11, 0.4.12, 0.4.13, 0.4.14, 0.4.15, 0.4.16, 0.4.17, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.4, 0.5.5, 0.5.6, 0.5.7, 0.5.8, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.7.0, 0.6.7, 0.6.8, 0.7.1, 0.7.3, 0.7.4, 0.7.5, 0.7.6, 0.7.7, 0.7.8, 0.7.9, 0.7.10, 0.7.11, 0.8.0, 0.8.1, 0.9.1, 0.10.0, 0.10.1, 0.10.2, 0.10.3, 0.10.4, 0.10.5, 0.10.6, 0.10.7, 0.11.0, 0.12.0, 0.12.2, 0.12.3, 0.13.0, 0.13.1, 0.13.2, 0.13.3, 0.14.0, 0.14.2, 0.14.3, 0.14.4, 0.15.0, 0.15.1, 0.15.2, 0.15.3, 0.15.4, 0.15.5, 0.16.0, 0.16.1, 0.16.2, 0.16.3, 0.18.0, 0.18.1, 0.18.2, 0.18.3, 0.18.4, 0.18.5, 0.18.6, 0.18.7, 1.0.1, 1.0.2, 1.0.3, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.2.4, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.3.6, 2.3.8, 2.3.9, 2.3.10, 2.3.11, 2.3.12, 2.3.13, 3.0.0-beta1, 3.0.0-beta2, 3.0.0, 3.0.1, 3.0.3, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.1.0, 3.1.1, 3.1.2, 3.2.1, 3.2.2, 3.2.3, 3.2.4, 3.2.5, 3.2.6, 3.2.7, 3.2.8, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.4.0, 3.4.1, 3.4.2, 4.0.0, 3.4.3, 4.0.1, 4.0.3, 4.0.4, 4.1.0, 4.1.1, 4.1.2, 4.2.0, 4.2.1, 5.0.0, 6.0.0-0, 6.0.0-1, 6.0.0-2, 6.0.0-3, 6.0.0-4, 6.0.0-5, 6.0.0-6, 6.0.0-7, 6.0.0-8, 6.0.0-9
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/james.brooks/Code/BlueBayTravel/Warehouse/npm-debug.log
```

**Comment by PeterDaveHello** [comment]
I can't not reopen this lol

**Comment by jbrooksuk** [comment]
Done :)

File: package.json (hunk 3 above)
Comment (PeterDaveHello) [inline package.json]: It's the latest stable version, unless you want 6.0.0.-0

**Comment by PeterDaveHello** [comment]
Updated.

**Comment by jbrooksuk** [comment]
Thank you :)
